include/ur_api.h

```diff
@@ -2929,7 +3002,8 @@ typedef enum ur_usm_alloc_info_t {
 UR_APIEXPORT ur_result_t UR_APICALL
 urUSMHostAlloc(
     ur_context_handle_t hContext, ///< [in] handle of the context object
-    ur_usm_mem_flags_t *pUSMFlag, ///< [in] USM memory allocation flags
+    ur_usm_desc_t *pUSMDesc,      ///< [in][optional] USM memory allocation descriptor
+    ur_usm_pool_handle_t pool,    ///< [in][optional] Pointer to a pool created using urUSMPoolCreate
     size_t size,                  ///< [in] size in bytes of the USM memory object to be allocated
     uint32_t align,               ///< [in] alignment of the USM memory object
```

Comment:
That makes sense, we would group all 'optional' parameters into one structure. However, I think it would be better to do in a separate PR. This would require changing some of the validation code I belive and I don't want to cause too many conflicts with: https://github.com/oneapi-src/unified-runtime/pull/303